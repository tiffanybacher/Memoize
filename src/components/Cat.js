import React, { Component } from 'react';

class Cat extends Component {
  render() {
    return (
      <div className="cat-banner">
        <div className="main">
          <span className="stand"></span>
          <div className="cat">
            <div className="body"></div>
            <div className="head">
              <div className="ear"></div>
              <div className="ear"></div>
            </div>
            <div className="face">
              <div className="nose"></div>
              <div className="whisker-container">
                <div className="whisker"></div>
                <div className="whisker"></div>
              </div>
              <div className="whisker-container">
                <div className="whisker"></div>
                <div className="whisker"></div>
              </div>
            </div>
            <div className="tail-container">
              <div className="tail">
                <div className="tail">
                  <div className="tail">
                    <div className="tail">
                      <div className="tail">
                        <div className="tail">
                          <div className="tail"></div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    );
  }
}
  
export default Cat;

// HTML by Anastasia Goodwin on Code Pen
