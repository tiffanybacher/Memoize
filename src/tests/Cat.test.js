import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import Cat from '../components/Cat';

describe('Cat', () => {
  let wrapper;
  
  beforeEach(() => {
    wrapper = shallow(
      <Cat />
    );
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
});