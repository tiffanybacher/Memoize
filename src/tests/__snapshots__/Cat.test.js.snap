// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cat should match snapshot 1`] = `
<div
  className="cat-banner"
>
  <div
    className="main"
  >
    <span
      className="stand"
    />
    <div
      className="cat"
    >
      <div
        className="body"
      />
      <div
        className="head"
      >
        <div
          className="ear"
        />
        <div
          className="ear"
        />
      </div>
      <div
        className="face"
      >
        <div
          className="nose"
        />
        <div
          className="whisker-container"
        >
          <div
            className="whisker"
          />
          <div
            className="whisker"
          />
        </div>
        <div
          className="whisker-container"
        >
          <div
            className="whisker"
          />
          <div
            className="whisker"
          />
        </div>
      </div>
      <div
        className="tail-container"
      >
        <div
          className="tail"
        >
          <div
            className="tail"
          >
            <div
              className="tail"
            >
              <div
                className="tail"
              >
                <div
                  className="tail"
                >
                  <div
                    className="tail"
                  >
                    <div
                      className="tail"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
